cppForSwig/cryptopp/configure.ac

```diff
@@ -88,11 +85,11 @@ AC_COMPILE_IFELSE(
 [CLANG=yes], [CLANG=no])
 AC_LANG_POP([C++])
 
-AC_MSG_RESULT([$CLANG])
+AC_MSG_RESULT([clang has been detected: $CLANG])
 AC_SUBST([CLANG])
 
 AM_CONDITIONAL([CRYPTOPP_HAVE_CLANG], [test x$CLANG = xyes])
-AM_CONDITIONAL([CRYPTOPP_HAVE_GCC], [test $CXX == g++])
+AM_CONDITIONAL([CRYPTOPP_HAVE_GCC], [test "x$CXX" = "xg++"])
```

Comment:
why is there an assignment operation in a test statement?

Reply:
I was trying to make some warnings go away when running ./configure. Looks like I goofed and really ought to rewrite the line above.

Reply:
Nevermind. "test" means it's not an assignment operation. You can check [here](https://www.gnu.org/software/automake/manual/html_node/Usage-of-Conditionals.html).